tests/conftest.py

```diff
@@ -45,6 +46,24 @@ def pytest_addoption(parser) -> None:
         action="store_true",
         default=None,
         help="Run private browsing test",
+    ),
+    parser.addoption(
+        "--experiment-slug",
+        action="store",
+        default=None,
+        help="Experiment slug from Experimenter",
+    ),
+    parser.addoption(
+        "--experiment-server",
+        action="store",
+        default="prod",
+        help="The server where the experiment is located, either stage or prod",
```

Comment:
Should this have something like this?
```py
choices=("prod", "stage"),
```

I'm just guessing based on my quick skim of https://docs.pytest.org/en/7.1.x/example/simple.html, but it might give us nice validation and errors like:

```sh
pytest: error: argument --experiment-server: invalid choice: 'production' (choose from 'prod', 'stage')
```

Reply:
Nice find!